internal/provider/inventory_data_source.go

```diff
@@ -59,8 +59,8 @@ func (d *InventoryDataSource) Schema(_ context.Context, _ datasource.SchemaReque
 			},
 			"variables": schema.StringAttribute{
 				Computed:    true,
-				CustomType:  jsontypes.NormalizedType{},
-				Description: "Variables of the inventory",
+				CustomType:  customtypes.AAPCustomStringType{},
+				Description: "Variables of the inventory. Must be provided as either a JSON or YAML string.",
```

Comment:
Since this is a data source, the user isn't providing this attribute. I would either revert this change or have it say something like "Will be either JSON or YAML string depending on how the variables were entered into AAP."

Reply:
I modified the description with your suggestion.